src/screens/CreatePost/index.js

```diff
@@ -71,6 +71,7 @@ import {getSpecificCache} from '../../utils/cache';
 import {getUrl, isContainUrl} from '../../utils/Utils';
 import {getUserId} from '../../utils/users';
 import {requestCameraPermission, requestExternalStoragePermission} from '../../utils/permission';
+import {Context} from '../../context';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `Context` is added but it's not clear from the provided context what this Context contains. Make sure that the name of the Context is descriptive enough to understand its purpose.
```diff
+ import {Context} from '../../context';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->